TTHAnalysis/python/analyzers/MT2Skimmer.py

```diff
@@ -29,12 +29,17 @@ def process(self, event):
         self.counters.counter('events').inc('all events')
 
         
-        ret = False 
-        
+        nElectrons10 = sum([l.pt() > 10 and abs(l.pdgId()) == 11 for l in event.selectedLeptons])
+        nMuons10 = sum([l.pt() > 10 and abs(l.pdgId()) == 13 for l in event.selectedLeptons])
+        nPFLep5LowMT = event.nPFLep5LowMT
+        nPFHad10LowMT = event.nPFHad10LowMT
+        nLep = nElectrons10 + nMuons10 + nPFLep5LowMT + nPFHad10LowMT
         ht = event.htJetXj10l5t
         nJet30 = sum([j.pt() > 30 for j in event.cleanJets])
         mt2= event.mt2_Xj
         met_pt = event.met.pt()
+        diffMetMht = event.diffMetMht
```

Comment:
Hi Bruno. 

It looks like I picked up the wrong variable. Do you know what is the difference between the two ? I correct it now, but I'll not launch a new tree production unless you tell me this is a showstopper. It affects only the QCD skimmed files, anyhow.